products/template/recipe.yml

```diff
@@ -0,0 +1,24 @@
+name: Template DB
```

Comment:
I think that for this, it might be nice to use some very, very small datasets (as opposed to Pluto). And Would be nice to test resolution of missing versions to latest, so maybe ✂️ one of the versions keys?

Reply:
see https://github.com/NYCPlanning/data-engineering/pull/231/commits/a533e64051b0656b92ed4ff9db7becf8ce4eadf0